node/core/candidate-validation/src/lib.rs

```diff
@@ -68,6 +69,34 @@ const PVF_EXECUTION_RETRY_DELAY: Duration = Duration::from_secs(3);
 #[cfg(test)]
 const PVF_EXECUTION_RETRY_DELAY: Duration = Duration::from_millis(200);
 
+/// Default PVF timeouts. Must never be changed! Use executor environment parameters in
+/// `session_info` pallet to adjust them
+
+/// For prechecking requests, the time period after which the preparation worker is considered
+/// unresponsive and will be killed.
+// NOTE: If you change this make sure to fix the buckets of `pvf_preparation_time` metric.
```

Comment:
These comments should probably be removed/updated in light of the "Must never be changed!" warning above. 😄

Reply:
You're right, I copied all the sensible comments to `enum PvfTimeoutType` and forgot to remove them here. Thank you!